README.md

```diff
@@ -83,6 +83,9 @@ To use this library, you describe the special behavior (if any) that resources o
 
 -  <a name="info"></a>`info` (optional): this allows you to provide extra information about the resource that will be included in the documentation. Available properties are `"description"` (a string describing what resources of this type are) and `"fields"`. `"fields"` holds an object in which you can describe each field in the resource (e.g. listing validation rules). See the [example implemenation](https://github.com/ethanresnick/json-api-example/blob/master/src/resource-descriptions/schools.js) for more details.
 
+## Filtering
```

Comment:
Thanks! This whole section was much needed, and it looks great. 

One thing, though, is that I've been a bit hesitant to document the support for Mongo query operators, because this support may be removed in future versions of the library (because such operators could open up security vulnerabilities around [nosql injection](https://www.owasp.org/index.php/Testing_for_NoSQL_injection) and their syntax is database-specific, whereas the final built-in filter set would ideally be backend agnostic). I'm fine mentioning them, though, as long as there's also a sentence or two warning that they might be removed. Do you think you could add that?